misc-utils/lsclocks.c

```diff
@@ -205,6 +212,38 @@ static clockid_t parse_clock(const char *name)
 	errx(EXIT_FAILURE, _("Unknown clock: %s"), name);
 }
 
+static int64_t get_namespace_offset(const char *name)
+{
+	char *tokstr, *buf, *saveptr, *line, *space;
+	uint64_t ret;
+	int fd;
+
+	fd = open(_PATH_PROC_TIMENS_OFF, O_RDONLY);
+	if (fd == -1)
+		err(EXIT_FAILURE, _("Could not open " _PATH_PROC_TIMENS_OFF));
```

Comment:
This is probably a gettext unfriendly way :-)  `_("Could not open %s"), _PATH_PROC_TIMENS_OFF` is already translated.